class ValidationError(Exception):
    pass

class NotFoundError(Exception):
    pass

class ConflictError(Exception):
    pass